src/Microsoft.DotNet.Build.Tasks/PackageFiles/crossgen.sh

```diff
@@ -18,7 +18,7 @@ restore_crossgen()
 
     __pjDir=$__toolsDir/crossgen
     mkdir -p $__pjDir
-    echo "<Project Sdk=\"Microsoft.NET.Sdk\"><PropertyGroup><TreatWarningsAsErrors>false</TreatWarningsAsErrors><NoWarn>$(NoWarn);NU1605;NU1103</NoWarn><TargetFramework>netcoreapp2.0</TargetFramework><DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences><RuntimeIdentifiers>$__packageRid</RuntimeIdentifiers></PropertyGroup><ItemGroup><PackageReference Include=\"Microsoft.NETCore.App\" Version=\"$__sharedFxVersion\" /></ItemGroup></Project>" > "$__pjDir/crossgen.csproj"
+    echo "<Project Sdk=\"Microsoft.NET.Sdk\"><PropertyGroup><DisableImplicitNuGetFallbackFolder>false</DisableImplicitNuGetFallbackFolder><TreatWarningsAsErrors>false</TreatWarningsAsErrors><NoWarn>$(NoWarn);NU1605;NU1103</NoWarn><TargetFramework>netcoreapp2.0</TargetFramework><DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences><RuntimeIdentifiers>$__packageRid</RuntimeIdentifiers></PropertyGroup><ItemGroup><PackageReference Include=\"Microsoft.NETCore.App\" Version=\"$__sharedFxVersion\" /></ItemGroup></Project>" > "$__pjDir/crossgen.csproj"
```

Comment:
Doesn't `$(NoWarn)` in `<NoWarn>$(NoWarn);NU1605;NU1103</NoWarn>` cause a bug because it's in double quotes, and the shell tries to get the stdout from the command `NoWarn` (which doesn't exist)?

Reply:
I didn't see these failures, and by lookign at the actual content of the .csproj it looks like we do keep $(NoWarn)

Reply:
On my OSX box, this comes to `<NoWarn>;NU1605;NU1103</NoWarn>`:

```
<Project Sdk="Microsoft.NET.Sdk"><PropertyGroup><DisableImplicitNuGetFallbackFolder>false</DisableImplicitNuGetFallbackFolder><TreatWarningsAsErrors>false</TreatWarningsAsErrors><NoWarn>;NU1605;NU1103</NoWarn><TargetFramework>netcoreapp2.0</TargetFramework><DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences><RuntimeIdentifiers>osx-x64</RuntimeIdentifiers></PropertyGroup><ItemGroup><PackageReference Include="Microsoft.NETCore.App" Version="2.0.3" /></ItemGroup></Project>

```

Reply:
you are right, I'll take a look and fix it, did you log a bug for it?

Reply:
I didn't. Thanks!